frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/utils/fileType.js

```
@@ -1,9 +1,8 @@
 const getFileTypeFromFileStoreURL = (filestoreURL) => {
+  if (typeof filestoreURL === "object") {
+    if (filestoreURL !== null && filestoreURL.master.includes(".m3u8")) return "hls";
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)